storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -540,6 +549,20 @@ func (c *PrometheusConverter) addTimeSeriesIfNeeded(lbls []prompb.Label, startTi
 	}
 }
 
+// handleStartTime adds a zero sample 1 millisecond before ts iff startTs == ts.
+func (c *PrometheusConverter) handleStartTime(startTs, ts int64, value float64, labels []prompb.Label, settings Settings) {
+	if !settings.EnableCreatedTimestampZeroIngestion {
+		return
+	}
+	if value != 0 && startTs > 0 && startTs == ts {
+		// NB: We currently handle only start timestamps which are the same as the data point timestamp.
```

Comment:
I don't like this phrasing; I would say we handle all start timestamps, as long as we see the first point.
Suggest linking to the umbrella issue for people wanting more explanation, and leave it out of the code.

Reply:
@bboreham are you suggesting to make the comment _only_ link to the umbrella issue instead?

Reply:
I put the issue link in the comment, please let me know if you had something else in mind.